### PR TITLE
Updated project to use Composer for installing PHP QA Toolchain 

### PR DESCRIPTION
**Pull request by Johnstonian**
Added Composer as the install vehicle for PHP QA Toolchain as PEAR support has reached end of life.

[Review on Reviewable](https://reviewable.io/reviews/chad-thompson/vagrantpress/87)

### CODE REVIEW

**Comment by Johnstonian** [comment]
oops, I changed the forwarded port (to 1234) in Vagrantfile because it was clashing with other vagrant instances on my machine. This should be reverted back to port 8080.

File: Vagrantfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ Vagrant.configure("2") do |config|
   config.vm.box = "trusty64"
   config.vm.box_url = "http://cloud-images.ubuntu.com/vagrant/trusty/current/trusty-server-cloudimg-amd64-vagrant-disk1.box"
 
-  config.vm.network :forwarded_port, guest: 80, host: 8080
+  config.vm.network :forwarded_port, host:1234, guest: 80
```
Comment (battlemidget) [inline Vagrantfile]: Mind changing this back? Once thats done I'll merge it in.

Thanks!!

Comment (Johnstonian) [inline Vagrantfile]: ok, forwarded port changed back to 8080. thanks

**Comment by battlemidget** [comment]
Thanks!

**Pull request merged**